### DISCUSSION TITLE
Will it work with multiple channel audios?

### DISCUSSION THREAD

**Discussion by felixvoir** [WyattBlue/auto-editor#464] [1. Help] [answered]
Hi i record 3 different stereo audio channels with obs for my desktop, mic and daw. So will it work with only 1 channel or you will bring an option to select which audio channels are going to be referenced for cutting process?

For example i will choose only, Microphone channel and it will cut silences with video in only microphone speech sections and wont touch any other areas.

Thanks!

**Comment by wowie1012** [comment]
I have the same question! Hope this problem gets investigated

**Comment by WyattBlue** [comment, marked as answer]
Yes! Auto-Editor will look at all the audio streams by default. If you want to look at the first track only, do `--edit audio:stream=0`. If you want to look at tracks 1 and 2 but ignore all others, do `--edit "(or audio:stream=0 audio:stream=1)"
